torch/_inductor/codecache.py

```diff
@@ -180,15 +180,21 @@ def get_system() -> Dict[str, Any]:
         try:
             system: Dict[str, Any] = {
                 "device": {
-                    "name": torch.cuda.get_device_properties(
-                        torch.cuda.current_device()
-                    ).name,
+                    "name": None
                 },
                 "version": {
-                    "cuda": torch.version.cuda,
                     "triton": triton_version,
                 },
             }
+            device_properties = torch.cuda.get_device_properties(
+                torch.cuda.current_device()
+            )
+            if torch.version.cuda is not None:
+                system["device"]["name"] = device_properties.name
+                system["version"]["cuda"] = torch.version.cuda
+            else:
+                system["device"]["name"] = device_properties.gcnArchName,
+                system["version"]["hip"] = torch.version.hip
         except (AssertionError, RuntimeError):
             # If cuda is not installed, none of the above config is relevant.
```

Comment:
This is a little janky.. but it's an existing issue

Reply:
As a followup, why don't we just set name=gcnArchName on AMD? With regards to the version info, I don't know if it is better to put both under cuda (we lose the clear distinction that one is AMD and the other is Nvidia) or keep the same. For now we'll just land this